content/api-docs/entry/alerts.md

```diff
@@ -120,13 +120,13 @@ A post-update alert will be returned.
   "id": "<alertId>",
   "status": "OK",
   ...
-  "reason": <reason>,
-  "opendAt": <opendAt>,
+  "reason": "<reason>",
```

Comment:
since `"<alertId>",` has double quotations around it, I thought I wanted to be consistent and added quotations to string fields.